src/sorcha/sorcha.py

```diff
@@ -43,36 +43,6 @@
 # Author: Samuel Cornwall, Siegfried Eggl, Grigori Fedorets, Steph Merritt, Meg Schwamb
 
 
-def runAll(args, configs, pplogger=None):
-    if configs["ephemerides_type"] == "ar":
-        runLSSTSimulation(args, configs, pplogger)
-    runLSSTPostProcessing(args, pplogger)
-
-
-def runLSSTSimulation(args, configs, pplogger=None):
-    if pplogger is None:
-        if type(args) is dict:
-            pplogger = PPGetLogger(args["outpath"])
-        else:
-            pplogger = PPGetLogger(args.outpath)
-
-    if type(args) is dict:
-        try:
-            args = sorchaArguments(args, pplogger)
-        except Exception as err:
-            pplogger.error(err)
-            sys.exit(err)
-
-    try:
-        args.validate_arguments()
-    except Exception as err:
-        pplogger.error(err)
-        sys.exit(err)
-
-    configs = PPConfigFileParser(args.configfile, args.surveyname)
-    create_ephemeris(args, configs)
-
-
 def runLSSTPostProcessing(args, configs, pplogger=None):
```

Comment:
This function will no longer be only postprocessing, so I think it would be good to change runLSSTPostProcessing to  runLSSTSimulation in terms of name, if possible.

Reply:
Done!